ddtrace/contrib/tornado/application.py

```diff
@@ -19,6 +19,8 @@ def tracer_config(__init__, app, args, kwargs):
         'tracer': ddtrace.tracer,
         'default_service': 'tornado-web',
         'distributed_tracing': True,
+        'analytics': None,
```

Comment:
Env variables?

Reply:
At the moment, none of the tracer configurations for `tornado` support environment variables, so I tried to avoid adding new behavior. We should revisit this as we bring the integration into the new Config API.

Reply:
:+1: